src/main/java/pro/sky/telegrambot/listener/TelegramBotUpdatesListener.java

```diff
@@ -28,9 +44,42 @@ public void init() {
     public int process(List<Update> updates) {
         updates.forEach(update -> {
             logger.info("Processing update: {}", update);
-            // Process your updates here
+            if (update.message().text().equals(new String("/start"))) {
+                long chatId = update.message().chat().id();
+                SendResponse response = telegramBot.execute(new SendMessage(chatId, "Приветик!Пришли мне сообщение вида 01.01.2022 20:00 Сделать домашнюю работу, а я напомню тебе о нем в нужное время))"));
+            } else {
+                try {
+                    String message = update.message().text();
+                    Pattern pattern = Pattern.compile("([0-9\\.\\:\\s]{16})(\\s)([\\W+]+)");
+                    Matcher matcher = pattern.matcher(message);
+                    if (matcher.matches()) {
+                        String time = matcher.group(1);
+                        String text = matcher.group(3);
+                        int chatId = Math.toIntExact(update.message().chat().id());
+                        LocalDateTime date2 = LocalDateTime.parse(time, DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm"));
```

Comment:
```suggestion
                        LocalDateTime sendingDate = LocalDateTime.parse(time, DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm"));
```
лучше давать переменным более осмысленные имена